mica-core/src/main/resources/config/project-form/definition.json

```diff
@@ -31,7 +31,9 @@
               {
                 "key": "endDate",
                 "dateOptions": {
-                  "dateFormat": "dd-MM-yyyy"
+                  "dateFormat": "dd-MM-yyyy",
+                  "minDate": "startDate",
+                  "minDateIsRef": true
```

Comment:
minDate is from the datapicker add-on and minDateIsRef is a new addition (it's already in release 0.1.3)